ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ClassicProtocolSpecs.java

```diff
@@ -149,19 +149,29 @@
         .name("Agharta");
   }
 
-  public static ProtocolSpecBuilder<Void> aztlanDefinition(
+  public static ProtocolSpecBuilder<Void> phoenixDefinition(
       final Optional<BigInteger> chainId,
       final OptionalInt configContractSizeLimit,
       final OptionalInt configStackSizeLimit,
       final boolean enableRevertReason) {
+    final int contractSizeLimit =
+        configContractSizeLimit.orElse(MainnetProtocolSpecs.SPURIOUS_DRAGON_CONTRACT_SIZE_LIMIT);
     return aghartaDefinition(
             chainId, configContractSizeLimit, configStackSizeLimit, enableRevertReason)
-        .gasCalculator(AztlanGasCalculator::new)
+        .gasCalculator(IstanbulGasCalculator::new)
         .evmBuilder(
             gasCalculator ->
-                MainnetEvmRegistries.aztlan(gasCalculator, chainId.orElse(BigInteger.ZERO)))
+                MainnetEvmRegistries.istanbul(gasCalculator, chainId.orElse(BigInteger.ZERO)))
         .precompileContractRegistryBuilder(MainnetPrecompiledContractRegistries::istanbul)
-        .name("Aztlan");
+        .contractCreationProcessorBuilder(
+            (gasCalculator, evm) ->
+                new MainnetContractCreationProcessor(
+                    gasCalculator,
+                    evm,
+                    true,
+                    Collections.singletonList(MaxCodeSizeRule.of(contractSizeLimit)),
+                    1))
```

Comment:
This appears to be identical to the definition already inherited from Atlantis. Am I missing something, should something have changed or is this just not required?

Reply:
This is indeed the same as Atlantis. If this isn't specified here, will it automatically use the previous one defined?

Reply:
Yes.  You should only need to specify the changes between each milestone.  So the only time you'd have two milestones with the same declaration is if something changed and then changed back (you'd have to override the changed version back to the original).

Reply:
Ok, this makes sense, I didn't think in was required, but I was following the pattern in MainnetProtocolSpec where contractCreationProcessBuilder is defined for SpuriouDragon then with the same definition for Istanbul without any changes is between.  I wasn't sure why that was done, but I followed that pattern.  I'll remove it from Phoenix since it has already been defined.